plugins/org.hibernate.eclipse.console/src/org/hibernate/eclipse/console/actions/ExecuteQueryAction.java

```diff
@@ -60,33 +60,39 @@ public void runWithEvent(Event event) {
 	}
 
 	protected void execute(QueryEditor queryEditor) {
+		try {
 
-		ConsoleConfiguration cfg = queryEditor.getConsoleConfiguration();
-		if (cfg != null) {
-			//keep states of ConsoleConfiguration and HibernateExtension synchronized
-			if (!(cfg.isSessionFactoryCreated() && cfg.getHibernateExtension().isSessionFactoryCreated())) {
-				if (ProjectCompilerVersionChecker.validateProjectComplianceLevel(cfg)){
-					if (queryEditor.askUserForConfiguration(cfg.getName())) {
-						if (!(cfg.hasConfiguration() && cfg.getHibernateExtension().hasConfiguration())) {
-		    				try {
-		    					cfg.build();
-		    				} catch (Exception he) {
-		    					HibernateConsolePlugin.getDefault().showError(
-		    						HibernateConsolePlugin.getShell(), 
-		    						HibernateConsoleMessages.LoadConsoleCFGCompletionProposal_could_not_load_configuration +
-		    						' ' + cfg.getName(), he);
-		    				}
-						}
-						if (cfg.hasConfiguration() && cfg.getHibernateExtension().hasConfiguration()) {
-							cfg.buildSessionFactory();
-							queryEditor.executeQuery(cfg);
+			ConsoleConfiguration cfg = queryEditor.getConsoleConfiguration();
+			if (cfg != null) {
+				//keep states of ConsoleConfiguration and HibernateExtension synchronized
+				if (!(cfg.isSessionFactoryCreated() && cfg.getHibernateExtension().isSessionFactoryCreated())) {
+					if (ProjectCompilerVersionChecker.validateProjectComplianceLevel(cfg)){
+						if (queryEditor.askUserForConfiguration(cfg.getName())) {
+							if (!(cfg.hasConfiguration() && cfg.getHibernateExtension().hasConfiguration())) {
+			    				try {
+			    					cfg.build();
+			    				} catch (Exception he) {
+			    					HibernateConsolePlugin.getDefault().showError(
+			    						HibernateConsolePlugin.getShell(), 
+			    						HibernateConsoleMessages.LoadConsoleCFGCompletionProposal_could_not_load_configuration +
+			    						' ' + cfg.getName(), he);
+			    				}
+							}
+							if (cfg.hasConfiguration() && cfg.getHibernateExtension().hasConfiguration()) {
+								cfg.buildSessionFactory();
+								queryEditor.executeQuery(cfg);
+							}
 						}
 					}
+				} else {
+					queryEditor.executeQuery(cfg);
 				}
-			} else {
-				queryEditor.executeQuery(cfg);
-			}
+			} 
+		
+		} catch (Throwable t) {
```

Comment:
True, but as I already explained to Alexey, this has been there since ages and it is not the cause of this particular problem. Removing the Throwable has a lot more implications and needs to be done with care as it might affect other pieces of the Eclipse Hibernate Tools project. It's something that shouldn't be done in this branch IMO.